src/tools/pub/dependencies.md

```diff
@@ -78,54 +66,77 @@ Currently, Flutter is the only SDK that is supported.
 
 The syntax looks like this:
 
-{% prettify yaml tag=pre+code %}
+```yaml
 dependencies:
   flutter_driver:
     sdk: flutter
-    version: ^0.0.1
-{% endprettify %}
+```
 
 The identifier after `sdk:` indicates which SDK the package comes from.
 If it's `flutter`, the dependency is satisfiable as long as:
 
 * Pub is running in the context of the `flutter` executable
 * The Flutter SDK contains a package with the given name
-* That package's version matches the version constraint
 
 If it's an unknown identifier, the dependency is always considered unsatisfied.
 
 A package with an `sdk` dependency
```

Comment:
When do we remove documentation for old versions? Version 1.19 starts to be pretty niche...